packages/react-charting/src/components/AreaChart/AreaChart.base.tsx

```diff
@@ -33,6 +33,7 @@ import {
   domainRangeOfNumericForAreaChart,
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31525&commitId=a6c7dcecf8f791bb08d41bb95a040565f38cd41e&env=prod&clientType=FLUENTUI)


  <details><summary><b>Pivot - Overflow</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Removed|

</details>


  <details><summary><b>react-charting-LineChart</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Basic - Dark Mode.hover.chromium.png|120| Changed|
|react-charting-LineChart.Basic.hover.chromium.png|149| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>